content/en/agent/fleet_automation/_index.md

```diff
@@ -0,0 +1,87 @@
+
+With Fleet Automation, you can:
+- Reduce the time it takes to debug an issue by sending a remote flare straight from the [Fleet Automation][1] page.
+- Ensure you're running up-to-date Agents with the latest security fixes and feature enhancements by identifying outdated Agent versions.
+- Identify which Agents are using a particular API Key so you can safely rotate keys and disable old keys without impact.
+- View Agent and Agent integration configurations to help confirm deployment changes and troubleshoot any configuration related issues.
+
+To access Fleet Automation, click **Integrations** > [**Fleet Automation**][1].
+
+From the Fleet Automation page, you can gain insight into unmonitored Agents, Agents that need to be updated, or Agents that have integration issues. For each Agent, you can see:
+- The Agent version
+- Whether the Agent has any unconfigured or misconfigured integrations
+- The services that the Agent is monitoring
+- The Remote Configuration status of each Agent
+- The products that are enabled on the Agent
+
+Selecting an Agent gives you more information about it, including its configuration, connected integrations, and a support tab that you can use to send a remote flare.
+
+{{< img src="agent/fleet_automation/selected_agent.png" alt="An Agent's integration information," style="width:100%;" >}}
+
+## Configuring Fleet Automation
+
+Fleet Automation incorporates several Datadog features, which are all enabled automatically in Agents version 7.49/6.49 or later. To ensure you have access to all of the features, upgrade your Agent to version 7.49/6.49 or later.
+
+If you're using an older Agent, you might still be able to enable the following Datadog features individually:
+- **Remote Configuration**: For information on supported Agent versions and configuration steps, see [Enabling Remote Configuration][3].
+- **Agent configuration**: Agent version 7.39/6.39 or later is required to enable the Agent configuration tab. It is enabled by default in Agent versions 7.47.0/6.47.0 or later. To enable Agent configuration manually, set `inventories_configuration_enabled` in your [Agent configuration file][2] to `true`. Alternatively, use the `DD_INVENTORIES_CONFIGURATION_ENABLED` environment variable.
+- **Agent integration configuration**: Agent integration configuration is enabled by default on Agent versions 7.49/6.49 or later. To enable Agent integration configuration manually, set `inventories_checks_configuration_enabled` in your [Agent configuration file][2] to `true`. Alternatively, use the environment variable `DD_INVENTORIES_CHECKS_CONFIGURATION_ENABLED`.
+
+Datadog recommends upgrading your Agents regularly to make sure you have access to the latest features and security updates.
+
+## Send a remote flare
+
+Before you send a flare, make sure that [Remote Configuration is enabled on the selected Agent](#configuring-fleet-automation).
```

Comment:
```suggestion
Before you send a flare, make sure that Remote Configuration is [enabled](#configuring-fleet-automation) on the selected Agent.
```

Reply:
The length of the hyperlinked phrase is definitely a matter of personal preference. My preference is for shorter links.